{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 13,
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "import glob\n",
    "from sqlalchemy import create_engine"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "metadata": {
    "scrolled": true
   },
   "outputs": [],
   "source": [
    "all_files = glob.glob(\"*.csv\")\n",
    "engine = create_engine('mysql://root:*****@localhost:3306/stock_db')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "metadata": {},
   "outputs": [
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'AAPL' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'AXP' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'BA' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'CAT' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'CSCO' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'CVX' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'DIS' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'DWDP' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'GS' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'HD' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'IBM' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'INTC' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'JNJ' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'JPM' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'KO' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'MCD' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'MMM' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'MRK' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'MSFT' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'NKE' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'PFE' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'PG' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'TRV' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'UNH' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'UTX' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'V' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'VZ' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n"
     ]
    },
    {
     "name": "stderr",
     "output_type": "stream",
     "text": [
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'WBA' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'WMT' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n",
      "C:\\Users\\roman\\Anaconda3\\envs\\PythonData\\lib\\site-packages\\pandas\\io\\sql.py:1144: UserWarning: The provided table name 'XOM' is not found exactly as such in the database after writing the table, possibly due to case sensitivity issues. Consider using lower case table names.\n",
      "  warnings.warn(msg, UserWarning)\n"
     ]
    }
   ],
   "source": [
    "for file in all_files:\n",
    "    words = file.split('-')\n",
    "    name = words[2].split('.')\n",
    "    df = pd.read_csv(file)\n",
    "    df.to_sql(name=name[0], con=engine)"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.6.6"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 2
}
